src/db/emoji_logging.rs

```diff
@@ -27,14 +27,12 @@ impl Db {
         emoji_name: impl AsRef<str>,
         emoji: &EmojiId,
     ) -> Result<EmojiData> {
-        let mut data = self.get_emoji_usage(emoji).await?;
         let mut conn = self.pool.acquire().await?;
         let emoji_str = emoji_name.as_ref();
-        data.reactions += count;
-        let num = data.reactions as i64;
         let id = emoji.0 as i64;
-        sqlx::query!("insert into emojis (emoji_id, emoji_name, reaction_usage) values (?1, ?2, ?3) on conflict(emoji_id) do update set reaction_usage=?3", id, emoji_str,num).execute(&mut conn).await?;
-        Ok(data)
+        let count = count as i64;
+        sqlx::query!("insert into emojis (emoji_id, emoji_name, reaction_usage) values (?1, ?2, ?3) on conflict(emoji_id) do update set reaction_usage=reaction_usage+?3", id, emoji_str, count).execute(&mut conn).await?;
```

Comment:
plox don't have these in single lines 😢  autoformat REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE